system_tests/openstack_handler.py

```diff
@@ -30,6 +30,9 @@
     BaseCloudifyInputsConfigReader)
 from cosmo_tester.framework.util import get_actual_keypath
 
+DEFAULT_SECURITY_GROUP_NAME = 'default'
+DEFAULT_EXTERNAL_NETWORK_NAME = 'Ext-Net'
+DEFAULT_SYSTEM_KEYPAIR_NAME = 'shared-systemt-tests-key'
```

Comment:
Once again, I would rather refrain from hard coding this value here even though it will only fallback to it.